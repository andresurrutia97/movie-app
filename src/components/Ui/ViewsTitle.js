import React from 'react';
import PropTypes from 'prop-types';

const ViewsTitle = ({ children }) => {
  return (
    <div className="text-2xl text-white md:text-6xl text-center md:text-left">
      {children}
    </div>
  );
};

ViewsTitle.propTypes = {
  children: PropTypes.string,
};

ViewsTitle.defaultProps = {
  children: '',
};

export default ViewsTitle;
