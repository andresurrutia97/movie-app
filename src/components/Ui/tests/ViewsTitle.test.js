import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import ViewsTitle from '../ViewsTitle';

describe('<ViewsTitle/>', () => {
  const component = <ViewsTitle>Test</ViewsTitle>;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should coantain a children text equal to "Test"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toBe('Test');
  });
});
