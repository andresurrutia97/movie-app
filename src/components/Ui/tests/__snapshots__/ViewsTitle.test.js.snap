// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewsTitle/> should render correctly  1`] = `
<div
  className="text-2xl text-white md:text-6xl text-center md:text-left"
>
  Test
</div>
`;
